### PR TITLE
Quantifying translocated gasdermin across treatments, doses, and neighbors

### PR DESCRIPTION
**Pull request by gwaybio**
Also adding plotnine to env

### CODE REVIEW

File: 0.wave1_data/4.analyze_wave1/nbconverted/gasdermin_intensity.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        ]
+    ) = "low"
+
+    # Create variable for translocation ratio
+    cp_df = (
+        cp_df
+        .assign(
+            translocation_ratio = (
+                cp_df.TranslocatedNuclei_Intensity_IntegratedIntensity_CorrGasderminD / 
+                cp_df.DilatedNuclei_Intensity_IntegratedIntensity_CorrGasderminD
```
Comment (jenna-tomkinson) [inline 0.wave1_data/4.analyze_wave1/nbconverted/gasdermin_intensity.py:75]: This is probably a very easy question, but how do these features divided represent a translocation ratio?

**Review by jenna-tomkinson — APPROVED** [review]
Just one small question to help me understand. LGTM otherwise and I am excited to discuss these with you!

File: 0.wave1_data/4.analyze_wave1/nbconverted/gasdermin_intensity.py (hunk 1 above)
Comment (gwaybio) [reply 0.wave1_data/4.analyze_wave1/nbconverted/gasdermin_intensity.py:75]: Based on the figure (https://github.com/WayScience/Interstellar_Project/blob/main/0.wave1_data/2.cellprofiler_analysis/figures/dilation_ex.png) the translocated nuclei object is our "cytoplasm" (nuclei excluded) and dilated nuclei object is our "cells" (nuclei included).

My thinking was that the ratio of "cytoplasm" gasdermin over total gasdermin represents the proportion of translocated gasdermin per cell.

I think this makes sense, but please correct me if I'm getting mixed up!

Comment (gwaybio) [reply 0.wave1_data/4.analyze_wave1/nbconverted/gasdermin_intensity.py:75]: I'm going to merge for now, but I can make changes later if needed.

**Pull request merged**